api/webscraper/nyserda_scraper.py

```diff
@@ -0,0 +1,98 @@
+                      'size': item.get('bid_quantity_mwh', None),
+                      'developer': item.get('developer', None),
+                      'proposed_cod': item.get('year_of_delivery_start_date', None),
+                      'county': item.get('county_province', None),
+                      'region': item.get('redc', None),
+                      'zipcode:': item.get('zip_code', None),
+                      'latitude': item.get('georeference')['coordinates'][0] if item.get('georeference', None) is not None else None,
+                      'longitude': item.get('georeference')['coordinates'][1] if item.get('georeference', None) is not None else None,                   'data_through_date': item.get('data_through_date', None),
+      }
+      filtered_list.append(project_dict)
+
+  with open('api/webscraper/nyserda_large.json', 'w') as file:
+      json.dump(filtered_list, file, indent=4)
+      file.write('\n')
+
+
+'''
+This scrapes data from the NYSERDA Statewide Distributed Solar Projects database.
+We filter for specific columns from the database's API and save them to a json file.
+https://data.ny.gov/Energy-Environment/Statewide-Distributed-Solar-Projects-Beginning-200/wgsj-jt5f/about_data
+
+geocode_lat_long is a helper util function that uses the google maps geocoding api to get the estimated
+latitude and longitude of a project based on the town
+'''
+nyserda_small_response = requests.get('https://data.ny.gov/resource/wgsj-jt5f.json')
+if nyserda_small_response.status_code != 200:
+  raise ValueError('Request to NYSERDA failed. Status code: %d\n%s' % (nyserda_small_response.status_code, nyserda_small_response.text))
+else:
+  small_data = nyserda_small_response.json()
+  small_list = []
+
+  for item in small_data:
+    if check_status(item.get('project_status', None)) != 'Cancelled':
+      if item.get('city_town', None) is not None:
+        lat, long = geocode_lat_long(f'{item.get('city_town')}, NY')
+      else:
+        lat, long = None, None
+      project_dict = {'project_name': item.get('project_id', None), # small data set only has project_id
+                      'project_status': check_status(item.get('project_status', None)), # missing
+                      'renewable_energy_technology': 'Solar',
+                      # 'size': item.get('bid_quantity_mwh', None), # what to use for small solar data?
+                      'developer': item.get('developer', None),
+                      'proposed_cod': item.get('interconnection_date', None),
+                      'county': item.get('county', None),
+                      'region': item.get('redc', None), # missing
+                      'zipcode:': item.get('zip', None),
+                      'latitude': lat,
+                      'longitude': long,
+                      'data_through_date': item.get('data_through_date', None),
+      }
+      small_list.append(project_dict)
```

Comment:
I am checking on the missing / ambiguous data and will have you fix anything once we get confirmation from the npo.